app/clock.hoon

```diff
@@ -0,0 +1,78 @@
+/+  *server
+/=  tile-js
+  /^  octs
+  /;  as-octs:mimes:html
+  /:  /===/app/clock/js/tile
+  /|  /js/
+      /~  ~
+  ==
+=,  format
+::
+|%
+:: +move: output effect
+::
++$  move  [bone card]
+:: +card: output effect payload
+::
++$  poke
+  $%  [%noun [@tas path @t]]
+  ==
+::
++$  card
+  $%  [%poke wire dock poke]
+      [%http-response =http-event:http]
+      [%connect wire binding:http-server term]
+      [%diff %json json]
+  ==
+::
+--
+::
+|_  [bol=bowl:gall ~]
+::
+++  this  .
+::
+++  bound
+  |=  [wir=wire success=? binding=binding:http-server]
+  ^-  (quip move _this)
+  [~ this]
+::
+++  prep
+  |=  old=(unit ~)
+  ^-  (quip move _this)
+  =/  lismov/(list move)  %+  weld
```

Comment:
`%+  weld` should start a new line, with appropriate backstep thereafter.

Reply:
fixed